src/Entity/ConfigEntity.php

```diff
@@ -28,6 +28,10 @@ class ConfigEntity implements EntityInterface
     #[ORM\Column(type: Types::BOOLEAN, nullable: true)]
     private ?bool $threeDSecureEnforced = null;
 
+    #[Groups(groups: ['admin-write'])]
+    #[ORM\Column(type: Types::STRING, length: 255, nullable: true)]
```

Comment:
I guess the postal code has a smaller length than 255 on braintree site?

Reply:
for the addresses it states:

> The postal code. Postal code must be a string of 4-9 alphanumeric characters, optionally separated by a dash or a space. Spaces and hyphens are ignored. 

I guess the same applies here? Think at least we should use a reasonable length of 20 chars